Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -291,12 +293,10 @@ public async Task RestartMainSiteIfNeeded(ITracer tracer, ILogger logger)
             {
                 return;
             }
-
             if (_settings.RecylePreviewEnabled())
```

Comment:
>if  [](start = 12, length = 3)

Not a good idea to introduce diffs unless it is intentional. This increases chances of merge conflicts.